scripts/what_changed.py

```diff
@@ -0,0 +1,345 @@
+            {
+                "area": "Native Port",
+                "regex" : ["^arch/posix"],
+                "counter": 0,
+                "labels": ["area: native port"]
+                },
+            {
+                "area": "X86",
+                "regex" : ["^arch/x86"],
+                "counter": 0,
+                "labels": ["area: X86"]
+                },
+            {
+                "area": "ARM",
+                "regex" : ["^arch/arm"],
+                "counter": 0,
+                "labels": ["area: ARM"]
+                },
+            {
+                "area": "Nios2",
+                "regex" : ["^arch/nios2"],
+                "counter": 0,
+                "labels": ["area: NIOS2"]
+                },
+            {
+                "area": "Xtensa",
+                "regex" : ["^arch/xtensa"],
+                "counter": 0,
+                "labels": ["area: Xtensa"]
+                },
+            {
+                "area": "RISCv32",
+                "regex" : ["^arch/riscv32"],
+                "counter": 0,
+                "labels": ["area: RISCv32"]
+                },
+            {
+                "area": "ARC",
+                "regex" : ["^arch/arc"],
+                "counter": 0,
+                "labels": ["area: ARC"]
+                },
+            {
+                "area": "Netowrking",
+                "regex" : ["^subsys/net"],
+                "counter": 0,
+                "labels": ["area: Networking"]
+                },
+            {
+                "area": "Logging",
+                "regex" : ["^subsys/logging"],
+                "counter": 0,
+                "labels": ["area: Logging"]
+                },
+            {
+                "area": "Shell",
+                "regex" : ["^subsys/shell"],
+                "counter": 0,
+                "labels": ["area: Shell"]
+                },
+            {
+                "area": "Console",
+                "regex" : ["^subsys/console"],
+                "counter": 0,
+                "labels": ["area: Console"]
+                },
+            {
+                "area": "Testsuite",
+                "regex" : ["^subsys/testsuite"],
+                "counter": 0,
+                "labels": ["area: Testing Suite"]
+                },
+            {
+                "area": "Settings",
+                "regex" : ["^subsys/settings"],
+                "counter": 0,
+                "labels": ["area: Settings"]
+                },
+            {
+                "area": "File System",
+                "regex" : ["^subsys/fs"],
+                "counter": 0,
+                "labels": ["area: File System"]
+                },
+            {
+                "area": "Storage",
+                "regex" : ["^subsys/storage"],
+                "counter": 0,
+                "labels": ["area: Storage"]
+                },
+            {
+                "area": "Bluetooth",
+                "regex" : ["^subsys/bluetooth"],
```

Comment:
could you change this one to just `*bluetooth*` to also catch their samples and tests?